### PR TITLE
Linted source

### PR DESCRIPTION
**Pull request by cbetta**
Work in progress to lint the source as well (tests are already linted and merged). Mostly fixable except for 2 issue. See below

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -616,11 +665,11 @@ exports.setHost = function(aHost) {
...
-}
+    niBasicEndpoint.host = aHost;
+    niStandardEndpoint.host = aHost;
+    applicationsEndpoint.host = aHost;
+};
 
 exports.setPort = function(aPort) {
-  port = aPort;
-}
+    port = aPort;
```
Comment (cbetta) [inline src/index.js:674]: ESLint says that `'port' is not defined` which is true. It also seems `port` is not used anywhere either. Should we remove this or properly implement this?

File: src/index.js [hunk 2]
```diff
@@ -3,22 +3,65 @@
 var querystring = require('querystring');
 
 var initialized = false;
```
Comment (cbetta) [inline src/index.js:5]: ESLint claims `'initialized' is defined but never used` even though it is on line 109. We should see and test if this actually works. It seems to me like JS is assigning a local variable on line 109, not a global one. But I'm not sure if this is an issue with ESLint or not.

File: src/index.js (hunk 1 above)
Comment (cbetta) [reply src/index.js:674]: @leggetter i think this is probably an example of some dead code linting has found.

**Comment by leggetter** [comment]
Let's kill this one for now.

**Pull request closed without merging**